app/services/back_end_service.rb

```diff
@@ -33,6 +33,13 @@ def create_user(user_params)
       )
     end
 
+    def create_event(params)
+      db_conn.post("/api/v1/users/#{params[:user_id]}/gym_memberships/#{params[:gym_membership_id]}/events") do |req|
+        req.params['activity'] = params[:activity]
+        req.params['date_time'] = params[:date_time]
```

Comment:
Idk if this is right... my recommendation would be to follow the only syntax we could get working for `create_user` (in prod)